nixos/platform/nix.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+let
+  location = lib.attrByPath [ "parameters" "location" ] "" config.flyingcircus.enc;
+  production = lib.attrByPath [ "parameters" "production" ] "" config.flyingcircus.enc;
+
+  nixPackage = if config.flyingcircus.nix.useUnstableNix
+    then pkgs.nixVersions.nix_2_25
+    else pkgs.nixVersions.nix_2_18;
+in {
+  options.flyingcircus = {
+    nix.useUnstableNix = lib.mkOption {
+      default = (location == "dev") || (location == "whq") || (production  == false);
+      defaultText = lib.literalExpression ''(location == "dev") || (location == "whq") || (production  == false)'';
```

Comment:
@osnyx you mentioned that you'd prefer 2.18 on staging first. This commit is the state I'd like to see in production at some point.

Would it make sense to you to push another commit on top that sets the default to `false` and adjusts the test accordingly? We could just revert the commit then when this can go to production.

Reply:
As we have 2 releasetest VMs per branch, I also suggest configuring one of them to use the stable nix for the time being.